modules/monixBio/src/weaver/monixbiocompat/suites.scala

```diff
@@ -13,10 +15,49 @@ trait BaseIOSuite { self: ConcurrentEffectSuite[Task] =>
   implicit def effect: ConcurrentEffect[Task] = IO.catsEffect(scheduler)
 }
 
+import cats.data.Chain
+import cats.effect.concurrent.Ref
+import cats.syntax.all._
+
+/**
+ * Individual test runner for Monix BIO's `IO[Throwable, A]` that properly handles unexpected errors,
+ * i.e. errors that occur in another channel.
+ */
+object Test {
+  def apply(
+      name: String,
+      f: Log[Task] => Task[Expectations]): Task[TestOutcome] =
+    for {
+      ref   <- Ref[Task].of(Chain.empty[Log.Entry])
+      start <- IO.clock.realTime(MILLISECONDS)
+      res <- IO
+        .defer(f(Log.collected[Task, Chain](ref)))
+        .map(Result.fromAssertion)
+        .redeemCauseWith(c => IO.raiseError(c.toThrowable), IO.now)
```

Comment:
Looks like it could be `redeemCause(c => Result.from(c.toThrowable), identity)` without `handleError` below

Reply:
Thanks, addressed in d2a587c05c8e3af3512ab6bd744fdf64ec0bb396.